MobiFlight/MidiBoard/MidiInputDefinition.cs

```diff
@@ -0,0 +1,43 @@
+﻿using Newtonsoft.Json;
+using Newtonsoft.Json.Converters;
+
+namespace MobiFlight
+{
+    public class MidiInputDefinition
+    {
+        public string Label;
```

Comment:
All these properties should get doc comments explaining what they are.

Reply:
I'll think about which ones really need a comment and are not self explanatory.